src/x509.c

```diff
@@ -1198,18 +1199,18 @@ int wolfSSL_X509V3_EXT_print(WOLFSSL_BIO *out, WOLFSSL_X509_EXTENSION *ext,
         {
             char isCa[] = "TRUE";
             char notCa[] = "FALSE";
-            if (XSNPRINTF(tmp, sz, "%*sCA:%s", indent, "",
-                                                    obj->ca ? isCa : notCa)
-                >= sz)
+            if ((tmp_len = XSNPRINTF(tmp, sizeof(tmp), "%*sCA:%s", indent, "",
```

Comment:
can we use a variable here instead of sizeof? Future fixes of small stack issues in separate PR's would be easier then.

Reply:
here in a couple other spots in this function

Reply:
yeah that's a good idea -- I'll revert those.  that notion did in fact occur to me, and is why I didn't do more of that...